mantle/kola/tests/upgrade/basic.go

```diff
@@ -242,6 +251,20 @@ func waitForUpgradeToVersion(c cluster.TestCluster, m platform.Machine, version
 	}
 }
 
+func waitForUpgradeToVersion(c cluster.TestCluster, m platform.Machine, version string) {
+	runFnAndWaitForRebootIntoVersion(c, m, version, func() {
+		// XXX: update to use https://github.com/coreos/zincati/issues/203
+		c.MustSSH(m, "sudo systemctl restart zincati.service")
+	})
+}
+
+func rebaseToStream(c cluster.TestCluster, m platform.Machine, ref, version string) {
+	runFnAndWaitForRebootIntoVersion(c, m, version, func() {
+		c.MustSSHf(m, "sudo rpm-ostree rebase %s", ref)
```

Comment:
> if we let `rpm-ostree` perform the reboot here, it's an additional logic path covered for free

I initially did do that (`rpm-ostree rebase --reboot ...`) but there isn't an easily exposed platform API right now for running a command which we know will cause a reboot (and possibly make SSH not exit cleanly). Hmm, I guess we could do `systemd-run ...` instead. OK did that!

> there is a possible-though-unlikely conflict here with Zincati (which is still running with the usual auto-updates logic). I'd be more comfortable if we stop that first to avoid any chances of spurious races

Good point, done!